mmv1/products/compute/Network.yaml

```diff
@@ -215,3 +215,12 @@ properties:
       If the field is not speficied, then a /48 range will be randomly allocated from fd20::/20 and returned via this field.
     immutable: true
     default_from_api: true
+  - !ruby/object:Api::Type::Enum
+    name: 'networkFirewallPolicyEnforcementOrder'
```

Comment:
Is this field used in any of the existing test?

Reply:
No I did not add an example for it. It looked like not all fields where covered. So I skiped it. Should I add it?

Reply:
I'd recommend to add at least 1 test to cover newly added fields.